<launch>
    <group ns="car1">
    
        <param name="robot_description" ... />
        <node name="joint_state_publisher" ... />
        <node name="robot_state_publisher" ... />
        <node name="urdf_spawner" ... />
        <rosparam file="..." command="load"/>
        <node name="controller_manager" ... />

        <node pkg="raceworld" type="lane.py" name="lane_suggester" output="screen" />

        <node pkg="raceworld" type="servo_commands.py" name="servo_commands" output="screen" >
            <param name="robot_name" value="car1"/>
            <remap from="ackermann_cmd_mux/output" to="lane_suggestion" />
        </node>
        
    </group>
</launch>
